.github/workflows/deploy.yml

```diff
@@ -82,23 +82,20 @@ jobs:
           echo "ecr_image_location=$ECR_IMAGE_LOCATION" >> $GITHUB_OUTPUT
 
       - name: checkout code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
-      - name: Set up Python
-        uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5
+      - name: check if changeset includes migrations
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
```

Comment:
any concerns around using a full third-party action in a pipeline that has a significant amount of AWS access?

Reply:
We could exclude it from renovate bumps by adding it to renovate.json in the root of the repo, yep, if we're happy with that: https://docs.renovatebot.com/configuration-options/#ignoredeps